### ISSUE TITLE
Memory Expansion Pak detected as a game cartridge in DS Classic Menu

### ISSUE THREAD

**Issue by lifehackerhansol** [DS-Homebrew/TWiLightMenu#1730]
**Console Configuration:** DS Lite, M3 DS Real, Nintendo DS Browser Memory Expansion Pak
**TWiLight Menu++ Version:** 36a07e54caa992c48927836fd21a3183e7d6c945
**TWiLight Menu++ Theme:** N/A

### Expected behaviour

Memory Expansion Pak is not detected as a game. On a DS Lite it shows up as an "optional cartridge" (translated. I don't know what it is in English)

### Actual behaviour

Shows up as a GBA game, and launches the pak as if it's a GBA game.

### Steps to reproduce

1. Insert Memory Expansion Pak
2. Run TWiLight Menu++ using your favourite flashcart and open DS Classic Menu
3. Start GBA game(?)

**Comment by Masamune3210** [comment]
Are you talking about a flashcard or the expansion pack? because the expansion pack isn't a game, its just a chunk of ram in a cart for things that need more ram than what the ds has onboard

**Comment by spellboundtriangle** [comment]
> Are you talking about a flashcard or the expansion pack? because the expansion pack isn't a game, its just a chunk of ram in a cart for things that need more ram than what the ds has onboard

That's the point of the issue

**Comment by Masamune3210** [comment]
Sorry, misread the issue and through you were trying to launch the pak as a game, my bad

**Comment by lifehackerhansol** [comment]
> Are you talking about a flashcard or the expansion pack? because the expansion pack isn't a game, its just a chunk of ram in a cart for things that need more ram than what the ds has onboard

Exactly, so the fact that it is recognized as a game in TWiLight is a bug.

**Comment by Epicpkmn11** [comment]
it wasn't actually recognizing the MEP as a game... it just wasn't checking at all lol
